.stickler.yml

```diff
@@ -0,0 +1,15 @@
+# add the linters you want stickler to use for this project
+linters:
+  eslint:
+    # indicate where is the config file for stylelint
+    config: './.eslintrc.json'
+
+# add the files here you want to be ignored by stylelint
+files:
+  ignore:
+      - 'dist/*'
+
+# PLEASE DO NOT enable auto fixing options
+# if you need extra support from you linter - do it in your local env as described in README for this config
+
+# find full documentation here: https://stickler-ci.com/docs
```

Comment:
- [ ] You can delete the stickler.yml file. You are already using Github actions, so using the stickler is not necessary.